docs/machine-learning/ml-problem-formulation-envisioning.md

```diff
@@ -34,6 +35,19 @@ or disproved - to guide the exploration of the data science team. Where possible
 
 Once an understanding of the problem domain has been established, it may be necessary to break down the overall problem into smaller, meaningful chunks of work to maintain team focus and ensure a realistic project scope within the given time frame.
 
+## Listening to the customer
+
+Throughout each session, we must keep in mind and proactively practice the exercise of listening to our customer. They are the center of gravity of any engagement, so we need to practice compassion by stepping into their shoes to feel their pain point. Therefore, it is important to always be mindful of their thoughts by asking leading questions to understand:
+
+* Who is the end user?
+* What is the current practice related to the business problem?
+* What's the performance of the current framework?
+* What are their pain points?
+* What is their toughest problem?
+* What is the state of the data used to build the solution?
```

Comment:
Maybe add "What would be an approach to solve this problem manually, if exists" or something similar. Just a thought.

Reply:
Do you mean to explore whether there is a non-ML solution to the problem?

Reply:
We believe the questions regarding the "current framework" cover this.